service/namespace.go

```diff
@@ -11,7 +14,12 @@ const resultChannel string = "Result"
 
 // namespace returns the namespace of the service.
 func (service *Service) namespace() []string {
-	return []string{service.Hash}
+	h, err := hex.DecodeString(service.Hash)
+	if err != nil {
+		panic(err)
+	}
+	sum := sha1.Sum([]byte(h))
```

Comment:
Is it really necessary to `DecodeString`?
Isn't it possible to just `sum := sha1.Sum([]byte(service.Hash))`?

Reply:
No it's not, but it will be more constant in the future when we will change sevice.Hash to []byte